core/lib/Thelia/Controller/Admin/ModuleController.php

```diff
@@ -362,4 +364,44 @@ public function installAction()
 
         return $this->render("modules");
     }
+
+    public function informationAction($module_id)
+    {
+        if (null !== $response = $this->checkAuth(AdminResources::MODULE, array(), AccessManager::VIEW)) {
+            return $response;
+        }
+
+        $status = 200;
+
+        if (null !== $module = ModuleQuery::create()->findPk($module_id)) {
+            $title = $module->setLocale($this->getSession()->getLang()->getLocale())->getTitle();
+
+            // Get the module descriptor
+            $moduleDescriptor = $module->getAbsoluteConfigPath() . DS . 'module.xml';
+
+            if (false !== $xmlData = @simplexml_load_file($moduleDescriptor)) {
+                // Transform the pseudo-array into a real array
+                $arrayData = json_decode(json_encode((array)$xmlData), 1);
+
+                $content = $this->renderRaw("ajax/module-information", [
+                    'moduleId' => $module_id,
+                    'moduleData' => $arrayData
+                ]);
+
+            } else {
+                $status = 404;
+
+                $content = $this->getTranslator()->trans(
+                    'Failed to load descriptor (module.xml) for module ID "%id".',
+                    [ '%id' => $module_id]
+                );
+            }
+
+        } else {
+            $title = $this->getTranslator()->trans('Error occured.');
```

Comment:
the 404 error is here and not in the previous. In the previous else it's more a 500 error.